### ISSUE TITLE
Write launch script

### ISSUE THREAD

**Issue by tanaes** [biocore/oecophylla#17]
Snakemake has an executable `snakemake` in the path with like a jillion command line parameters. 

In the cluster environment, it's much more efficient to use a launch script, which invokes `snakemake` and streamlines the choice of other parameters appropriate for our cluster. 

For example, here is the `launch.sh` script I have been using: 

```bash
#!/bin/bash

if [ "$#" -lt 1 ]; then
    echo "script.sh <out_dir (with config.yaml)> [additional snakemake arguments]*"
    exit
fi

outdir=$1

mkdir -p $outdir/cluster_logs

snakemake -j 16 --local-cores 4 -w 90 --cluster-config cluster.json --cluster "qsub -e {cluster.error} -o {cluster.output} -m n -l nodes=1:ppn={cluster.n} -l mem={cluster.mem}gb -l walltime={cluster.time}" --directory "$@"
```

This shell script is launching in a format appropriate for Barnacle (qsub/Torque), and choosing some options (`-w 90` to deal with filesystem latency, `-j 16` to only submit 16 jobs to queue at a time to be a good citizen) that make sense in that context. 

However, when I'm launching locally, this doesn't work. Instead, I use the following `launch_local.sh` script: 

```bash
#!/bin/bash

if [ "$#" -lt 1 ]; then
    echo "script.sh <out_dir (with config.yaml)> [additional snakemake arguments]*"
    exit
fi

outdir=$1

mkdir -p $outdir/cluster_logs

snakemake -j 16 --local-cores 4 -w 90 --cluster-config cluster.json --cluster "touch {cluster.output}; " --directory "$@"
```

In both of these cases, the specific rules and other parameters to be passed to snakemake are passed in $@. 

It would be *super nice* to have an installed python launch script in $PATH that streamlines this process, and maybe wraps alternative cluster submit options (specifically Torque for Barnacle, Slurm for Comet, and local execution).

**Comment by mortonjt** [comment]
It would be preferable to have this recognized by the setup.py, so that this can be run anywhere

Relevant to #24

**Comment by mortonjt** [comment]
For this one, I think we'll want to have both options to
- specify a folder and parse out all of the filenames accordingly (i.e. discriminate between forward/reverse)
- pass in a sample manifest specifying all sample prep information

**Comment by tkosciolek** [comment]
I'm also adding handling installation, e.g.
`oecophylla install` would install all defaults (i.e. conda environments) and
`oecophylla install --config-file install.txt` would install as specified by `install.txt`

**Comment by tanaes** [comment]
Fixed in #77 (for now, doesn't handle installation but moving to separate issue)
